### PR TITLE
Finalize Light Mode / Dark Mode (with new primitives)

### PR DESCRIPTION
**Pull request by grgia**
### Details

This PR updates both light and dark product color themes to match figma. We update the highlightBG color, add some remaining themed illustrations, and update the hover color.

### Fixed Issues

$ https://github.com/Expensify/App/issues/32131
$ https://github.com/Expensify/App/issues/32188
$ https://github.com/Expensify/App/issues/28817

### Tests

- [ ] Verify that no errors appear in the JS console
1. Go to Settings -> Workspaces -> Click a workspace without a bank account -> Bank account -> Add Manually 
- [ ] Verify that the correct check preview image is shown in:
- [ ] Light Mode + English Language
- [ ] Dark Mode + English Language
- [ ] Light Mode + Spanish Language
- [ ] Dark Mode + Spanish Language

Screenshot 2023-12-06 at 2 44 19 PMScreenshot 2023-12-06 at 2 44 07 PMScreenshot 2023-12-06 at 2 43 59 PMimage

2.  Go to Profile -> Click on Profile Avatar
- [ ] Verify that the border around the camera icon matches the background in both light and dark mode
imageimage

3. Open a chat -> type `@` and verify that the highlight color matches these mocks:
image
image

4. Visually QA Hover/Active/Regular LHN colors match this mock:
image
image

**DESIGN: HOW TO HELP QA:**
1. Open the most recent Adhoc Build on your favorite platform and log into your `expensifail.com`
2. To change themes, go to `/preferences/themes` and select `Light` and `Dark`. See below:

https://github.com/Expensify/App/assets/38015950/64c86ff8-b2fb-41bd-a8ac-a845cfdb7452

6. Take a screenshot of any incorrect colors and leave a comment with a screenshot and what changes need to be made

**IN DEV: HOW TO HELP QA:**
1. Use these statements in the console:

To switch themes in DEV:

```js
Onyx.merge("preferredTheme", 'dark');

Onyx.merge("preferredTheme", 'light');

```
To switch languages in DEV:

```js
Onyx.merge("preferredLocale", 'en');

Onyx.merge("preferredLocale", 'es');

```

### QA Steps

1. Go to Settings -> Workspaces -> Click a workspace without a bank account -> Bank account -> Add Manually 
- [ ] Verify that the correct check preview image is shown in:
- [ ] Dark Mode + English Language
- [ ] Dark Mode + Spanish Language
2.  Go to Profile -> Click on Profile Avatar
- [ ] Verify that the border around the camera icon matches the background in dark mode
3. Open a chat -> type `@` and verify that the highlight color matches these mocks:
image
### PR Author Checklist

- [X] I linked the correct issue in the `### Fixed Issues` section above
- [X] I wrote clear testing steps that cover the changes made in this PR
    - [X] I added steps for local testing in the `Tests` section
    - [X] I added steps for the expected offline behavior in the `Offline steps` section
    - [X] I added steps for Staging and/or Production testing in the `QA steps` section
    - [X] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [X] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I ran the tests on **all platforms** & verified they passed on:
    - [X] Android: Native
    - [X] Android: mWeb Chrome
    - [X] iOS: Native
    - [X] iOS: mWeb Safari
    - [X] MacOS: Chrome / Safari
    - [X] MacOS: Desktop
- [X] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [X] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [X] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [X] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] I verified that if a function's arguments changed that all usages have also been updated correctly
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

image

</details>

<details>
<summary>iOS: mWeb Safari</summary>

image

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

image
image

</details>

<details>
<summary>MacOS: Desktop</summary>

image
image

</details>

### CODE REVIEW

**Comment by grgia** [comment]
cc @dannymcclain @dubielzyk-expensify @shawnborton @chrispader for QA

**Comment by grgia** [comment]
cc @dannymcclain @dubielzyk-expensify @shawnborton @chrispader for some reason the latest ADHOC didn't include the last commit- looking into it 🙃

**Comment by dannymcclain** [comment]
Hahaha I kept looking and looking thinking that I had gone crazy! No worries.

**Comment by grgia** [comment]
Asked in engineering-chat: https://expensify.slack.com/archives/C03TQ48KC/p1701812296195999

**Comment by grgia** [comment]
@dannymcclain @shawnborton @dubielzyk-expensify 

Do a hard refresh and you should be able to load the theme page- https://32512.pr-testing.expensify.com/settings/preferences/theme

**Comment by shawnborton** [comment]
I'm still seeing the issue where the light mode LHN colors don't have the correct selected state and hover state, notice how they look the same:
CleanShot 2023-12-06 at 15 18 15@2x


But on dark mode, they are different:
CleanShot 2023-12-06 at 15 19 25@2x

**Comment by grgia** [comment]
@sobitneupane, just a note: desktop is broken on main, but the last ADHOC build includes all changes except for the LHN hover colors. I decided not to rebuild as I wanted to remove the temporary settings from this PR so we can merge. (Which allows you to switch themes in ADHOC)

**Comment by grgia** [comment]
Ready for final review @sobitneupane

**Comment by grgia** [comment]
I lied, now it's ready for final review @sobitneupane 🙏 
If you could please prioritize this, we are aiming to launch this feature in the next week :)

**Comment by grgia** [comment]
@sobitneupane I merged main for the desktop build fix :)

**Review by sobitneupane — COMMENTED** [review]
### Screenshots/Videos
<details>
<summary>Android: Native</summary>



<details>
<summary>Go to Settings -> Workspaces -> Click a workspace without a bank account -> Bank account -> Add Manually</summary>

 Light Mode + English Language
 
Screenshot 2023-12-07 at 18 22 47

 Dark Mode + English Language
 
Screenshot 2023-12-07 at 17 50 24

 Light Mode + Spanish Language
Screenshot 2023-12-07 at 18 22 57

 Dark Mode + Spanish Language
Screenshot 2023-12-07 at 17 52 31


 </details>
<details>
<summary>Go to Profile -> Click on Profile Avatar</summary>
Screenshot 2023-12-07 at 17 53 25
Screenshot 2023-12-07 at 18 25 46

</details>

<details>
<summary>Open a chat -> type @ and verify that the highlight color matches these mocks:</summary>
Screenshot 2023-12-07 at 17 54 03
Screenshot 2023-12-07 at 18 24 10

</details>
<details>
<summary> Visually QA Hover/Active/Regular LHN colors match this mock:</summary>


Screenshot 2023-12-07 at 17 57 55
Screenshot 2023-12-07 at 18 23 51


</details>

https://github.com/Expensify/App/assets/25876548/eed3e711-c9a8-41cc-a0a5-07d8339e2c52
</details>

<details>
<summary>Android: mWeb Chrome</summary>

<details>
<summary>Go to Settings -> Workspaces -> Click a workspace without a bank account -> Bank account -> Add Manually</summary>

 Light Mode + English Language
 


 Dark Mode + English Language
 

 Light Mode + Spanish Language

 Dark Mode + Spanish Language


 </details>
<details>
<summary>Go to Profile -> Click on Profile Avatar</summary>

</details>

<details>
<summary>Open a chat -> type @ and verify that the highlight color matches these mocks:</summary>


</details>
<details>
<summary> Visually QA Hover/Active/Regular LHN colors match this mock:</summary>


</details>
</details>
<details>

<summary>iOS: Native</summary>

<details>
<summary>Go to Settings -> Workspaces -> Click a workspace without a bank account -> Bank account -> Add Manually</summary>

 Light Mode + English Language
 
Screenshot 2023-12-07 at 18 19 55


 Dark Mode + English Language
 
Screenshot 2023-12-07 at 18 20 37


 Light Mode + Spanish Language
Screenshot 2023-12-07 at 18 20 05

 Dark Mode + Spanish Language

Screenshot 2023-12-07 at 18 20 20

 </details>
<details>
<summary>Go to Profile -> Click on Profile Avatar</summary>
Screenshot 2023-12-07 at 18 18 46
Screenshot 2023-12-07 at 18 20 49


</details>

<details>
<summary>Open a chat -> type @ and verify that the highlight color matches these mocks:</summary>
Screenshot 2023-12-07 at 18 19 03

Screenshot 2023-12-07 at 18 21 51

</details>
<details>
<summary> Visually QA Hover/Active/Regular LHN colors match this mock:</summary>

Screenshot 2023-12-07 at 18 17 14

Screenshot 2023-12-07 at 18 22 03

</details>
</details>

<details>
<summary>iOS: mWeb Safari</summary>

<details>
<summary>Go to Settings -> Workspaces -> Click a workspace without a bank account -> Bank account -> Add Manually</summary>

 Light Mode + English Language
 
Screenshot 2023-12-07 at 18 07 08

 Dark Mode + English Language
 
Screenshot 2023-12-07 at 18 08 17


 Light Mode + Spanish Language
Screenshot 2023-12-07 at 18 07 30


 Dark Mode + Spanish Language
Screenshot 2023-12-07 at 18 08 02



 </details>
<details>
<summary>Go to Profile -> Click on Profile Avatar</summary>
Screenshot 2023-12-07 at 18 08 31
Screenshot 2023-12-07 at 18 08 48

</details>

<details>
<summary>Open a chat -> type @ and verify that the highlight color matches these mocks:</summary>
Screenshot 2023-12-07 at 18 09 16
Screenshot 2023-12-07 at 18 09 43

</details>
<details>
<summary> Visually QA Hover/Active/Regular LHN colors match this mock:</summary>
Screenshot 2023-12-07 at 18 08 59
Screenshot 2023-12-07 at 18 09 34


</details>
https://github.com/Expensify/App/assets/25876548/0ecda5a3-379f-4b35-b1e8-6c1053b31e34

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<details>
<summary>Go to Settings -> Workspaces -> Click a workspace without a bank account -> Bank account -> Add Manually</summary>

 Light Mode + English Language
 
Screenshot 2023-12-07 at 17 20 52

 Dark Mode + English Language
 
Screenshot 2023-12-07 at 17 04 40

 Light Mode + Spanish Language
Screenshot 2023-12-07 at 17 21 23

 Dark Mode + Spanish Language
Screenshot 2023-12-07 at 17 17 42

 </details>
<details>
<summary>Go to Profile -> Click on Profile Avatar</summary>
Screenshot 2023-12-07 at 17 23 16
Screenshot 2023-12-07 at 17 23 36
</details>
<details>
<summary>Open a chat -> type @ and verify that the highlight color matches these mocks:</summary>
Screenshot 2023-12-07 at 17 25 16
Screenshot 2023-12-07 at 17 25 42
</details>
<details>
<summary> Visually QA Hover/Active/Regular LHN colors match this mock:</summary>
Screenshot 2023-12-07 at 17 26 44
Screenshot 2023-12-07 at 17 27 52

</details>

https://github.com/Expensify/App/assets/25876548/97d9afa7-5488-4df1-a24c-ad25f5463b34

https://github.com/Expensify/App/assets/25876548/3b16a9f6-fee5-40ec-8ceb-40a75cc83b20


</details>

<details>
<summary>MacOS: Desktop</summary>

<details>
<summary>Go to Settings -> Workspaces -> Click a workspace without a bank account -> Bank account -> Add Manually</summary>

 Light Mode + English Language
 

Screenshot 2023-12-07 at 18 13 43

 Dark Mode + English Language
 
Screenshot 2023-12-07 at 18 12 21


 Light Mode + Spanish Language
Screenshot 2023-12-07 at 18 13 28


 Dark Mode + Spanish Language
Screenshot 2023-12-07 at 18 12 50


 </details>
<details>
<summary>Go to Profile -> Click on Profile Avatar</summary>
Screenshot 2023-12-07 at 18 11 36
Screenshot 2023-12-07 at 18 11 51
Screenshot 2023-12-07 at 18 14 04

</details>

<details>
<summary>Open a chat -> type @ and verify that the highlight color matches these mocks:</summary>

Screenshot 2023-12-07 at 18 11 24
Screenshot 2023-12-07 at 18 14 15


</details>
<details>
<summary> Visually QA Hover/Active/Regular LHN colors match this mock:</summary>

https://github.com/Expensify/App/assets/25876548/7476d274-ffd6-4c40-975f-cbfea467d75c


https://github.com/Expensify/App/assets/25876548/fcd25a1c-80d4-4cf1-a43d-9dcb75f2b9ec


</details>

https://github.com/Expensify/App/assets/25876548/8c87f011-20fe-4cae-b974-f5b3b281feb6



https://github.com/Expensify/App/assets/25876548/17e4686e-dff6-498d-8fe3-d95a941b003d


</details>

**Review by sobitneupane — COMMENTED** [review]
Few issues that I noticed so far:

- Sometimes in mWeb/Safari and mWeb/Chrome the color of status bar (in light mode) changes back to dark. One way to reproduce is:

1. Go to any chat.
2. Press on the header to open the users profile page.
Screenshot 2023-12-07 at 18 31 30

- In IOS/native, Color of text in status bar is dark in dark mode
Screenshot 2023-12-07 at 18 37 33

- Similarly, in Android/native, Color of text in status bar is white in light mode.
Screenshot 2023-12-07 at 18 38 51

**Comment by grgia** [comment]
@sobitneupane since we're handling those in a follow up, is there anything else blocking from approval?

**Review by sobitneupane — APPROVED** [review]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

**Review by Gonals — APPROVED** [review]

File: src/styles/themes/default.ts [hunk 1]
```diff
@@ -76,10 +76,10 @@ const darkTheme = {
...
     ourMentionText: colors.green100,
     ourMentionBG: colors.green600,
-    tooltipSupportingText: colors.lightSupportingText,
-    tooltipPrimaryText: colors.lightPrimaryText,
-    skeletonLHNIn: colors.darkBorders,
-    skeletonLHNOut: colors.darkDefaultButton,
+    tooltipSupportingText: colors.productLight800,
+    tooltipPrimaryText: colors.productLight900,
+    skeletonLHNIn: colors.productDark400,
+    skeletonLHNOut: colors.productDark400,
```
Comment (situchan) [inline src/styles/themes/default.ts:82]: As these are same colors, it caused regression - #32719

Comment (situchan) [inline src/styles/themes/default.ts:82]:
```suggestion
    skeletonLHNOut: colors.productDark600,
```

**Pull request merged**